File: 2000-2999/2723/OpenJudge2886.java
import java.io.* ;
import java.util.* ;

public class Main {
    public static void main(String[] args) {
        Scanner cin = new Scanner(System.in) ;
        while (cin.hasNext()) {
            int n = cin.nextInt() ;
            int sum = 0 ;
            for (int i = 3; i <= n; i +=3 ) {
                sum += i ;
            }
            System.out.println(sum) ;
        }
    }
}

File: 2000-2999/OpenJudge2886.cpp
#include <iostream>

using namespace std ;

int main(int argc, char *argv[])
{
	int n ;
	while (cin >> n)
	{
		int sum = 0 ;
		for (int i = 0; i <= n; i += 3)
			sum += i ;
		cout << sum << endl ;
	}
	return 0 ;
}
